### PR TITLE
[MU4] fixed clicking on notation on HIDPI screens

### PR DESCRIPTION
**Pull request by RomanPudashkin**

### CODE REVIEW

File: mu4/notation/view/notationpaintview.cpp [hunk 1]
```diff
@@ -360,19 +346,22 @@ void NotationPaintView::dropEvent(QDropEvent* ev)
...
+    QPoint scaledPoint(point.x() * scale, point.y() * scale);
+
+    return m_matrix.inverted().map(scaledPoint);
 }
 
-QRect NotationPaintView::toLogical(const QRect& r) const
+double NotationPaintView::guiScale() const
 {
-    return m_matrix.inverted().mapRect(r);
+    return window()->screen()->devicePixelRatio();
```
Comment (igorkorsukov) [inline mu4/notation/view/notationpaintview.cpp:359]: we have `UiConfiguration::guiScaling`, I think better use it (and move this code there)

**Comment by Jojo-Schmitz** [comment]
rebase needed

File: mu4/notation/view/notationpaintview.cpp (hunk 1 above)
Comment (RomanPudashkin) [reply mu4/notation/view/notationpaintview.cpp:387]: This is a problem. Here need to get the scale of the current screen (on which a view is placed). I could not find any solution without using window() of QQuickPaintedItem

Comment (igorkorsukov) [reply mu4/notation/view/notationpaintview.cpp:387]: there is `IMainWindow` interface, which can be used in the UiConfiguration by injection

Comment (RomanPudashkin) [reply mu4/notation/view/notationpaintview.cpp]: fixed

**Review by igorkorsukov — APPROVED** [review]

**Pull request merged**